src/patternfly/components/TreeView/tree-view.scss

```diff
@@ -0,0 +1,198 @@
+      }
+
+      .pf-c-tree-view__link {
+        --pf-c-tree-view__link--PaddingLeft: var(--pf-c-tree-view__list--m-nested-level-2__list-item__link--PaddingLeft);
+      }
+    }
+  }
+
+  &.pf-m-nested-level-3 {
+    .pf-c-tree-view__list-item {
+      &.pf-m-expandable > .pf-c-tree-view__link {
+        --pf-c-tree-view__link--PaddingLeft: var(--pf-c-tree-view__list--m-nested-level-3__list-item--m-expandable__link--PaddingLeft);
+      }
+
+      .pf-c-tree-view__link {
+        --pf-c-tree-view__link--PaddingLeft: var(--pf-c-tree-view__list--m-nested-level-3__list-item__link--PaddingLeft);
+      }
+    }
+  }
+}
+// stylelint-enable
+
+.pf-c-tree-view__list-item {
+  &.pf-m-expanded {
+    > .pf-c-tree-view__link > .pf-c-tree-view__link-toggle-icon {
+      transform: rotate(var(--pf-c-tree-view__link--m-expanded__toggle-icon--Rotate));
+    }
+  }
+}
+
+.pf-c-tree-view__link {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  padding: var(--pf-c-tree-view__link--PaddingTop) var(--pf-c-tree-view__link--PaddingRight) var(--pf-c-tree-view__link--PaddingBottom) var(--pf-c-tree-view__link--PaddingLeft);
+  color: var(--pf-c-tree-view__link--Color);
+  text-align: left;
+  border: 0;
+
+  &.pf-m-current {
+    --pf-c-tree-view__link--Color: var(--pf-c-tree-view__link--m-current--Color);
+
+    font-weight: var(--pf-c-tree-view__link--m-current--FontWeight);
+  }
+
+  &.pf-m-first {
+    font-weight: var(--pf-c-tree-view__link--m-first--FontWeight);
+  }
+
+  &:focus {
+    background-color: var(--pf-c-tree-view__link--focus--BackgroundColor);
+  }
+
+  input {
+    margin-right: var(--pf-c-tree-view__link--input--MarginRight);
+  }
+
+  .pf-c-tree-view__link-count {
+    margin-left: var(--pf-c-tree-view__link-count--MarginLeft);
+
+    .pf-c-badge.pf-m-read {
+      --pf-c-badge--m-read--BackgroundColor: var(--pf-c-tree-view__link-count--c-badge--m-read--BackgroundColor);
+    }
+  }
+}
+
+.pf-c-tree-view__link-toggle-icon {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  min-width: var(--pf-c-tree-view__link-toggle-icon--MinWidth);
+  min-height: var(--pf-c-tree-view__link-toggle-icon--MinHeight);
+  padding-right: var(--pf-c-tree-view__link-toggle-icon--PaddingRight);
+  padding-left: var(--pf-c-tree-view__link-toggle-icon--PaddingLeft);
+}
+
+.pf-c-tree-view__link-text {
+  // @include pf-text-overflow;
+
+  @include pf-line-clamp("var(--pf-c-tree-view__link-text--max-lines)");
+
+  word-break: break-word;
+}
+
+.pf-c-tree-view__search {
+  padding: var(--pf-c-tree-view__search--PaddingTop) var(--pf-c-tree-view__search--PaddingRight) var(--pf-c-tree-view__search--PaddingBottom) var(--pf-c-tree-view__search--PaddingLeft);
+}
+
+.pf-c-tree-view__link-icon {
+  padding-right: var(--pf-c-tree-view__link-icon--PaddingRight);
+  color: var(--pf-c-tree-view__link-icon--Color);
+}
+
+.pf-c-tree-view__node {
```

Comment:
I'm not sure about the `__node` name :) In the app launcher, we named the `li` `__wrapper`. It doesn't seem like we need this `__node` element? Or is it needed for something? 

One thing it will help with is if we add a `__description` or something, and want to separate the item's top row of things from other things below it. Maybe this could be called `__main` or just `__row`?